odbc_interface.c

```diff
@@ -1958,7 +1970,11 @@ ODBC_INTERFACE RETCODE SQL_API
 SQLDescribeParam (SQLHSTMT StatementHandle,
 		  SQLUSMALLINT ParameterNumber,
 		  SQLSMALLINT * DataTypePtr,
+#if defined (_WINDOWS)
```

Comment:
If you look at SQLDescribeParam() defined in sqlext.h, ParameterSizePtr is of SQLULEN type.
Is it necessary to differentiate by OS?

SQLRETURN SQL_API SQLDescribeParam(
    SQLHSTMT           hstmt,
    SQLUSMALLINT       ipar,
    SQLSMALLINT           *pfSqlType,
    SQLULEN           *pcbParamDef,
    SQLSMALLINT           *pibScale,
    SQLSMALLINT           *pfNullable);

Reply:
I agree with your comment, but Linux gcc compiler do not allow such a thing with a reason of there is a conflict between function prototype and function definition in argument, **ParameterSizePtr**, and I do not make a side effect on Windows ODBC code. So, I modified it for non-Windows OS (Linux, AIX) by macro definition.